src/util/Vocabularies.ts

```diff
@@ -128,6 +128,11 @@ export const SOLID_HTTP = createUriAndTermNamespace('urn:npm:solid:community-ser
   'slug',
 );
 
+export const SOLID_META = createUriAndTermNamespace('urn:npm:solid:community-server:meta:',
+  // This identifier is used as graph for all metadata that is generated on the fly and should not be stored
+  'Generated',
+);
```

Comment:
1. Is this graph ever exposed? (I don't think it should.) The choice here is to use the _same_ graph name for metadata for all resources; another choice would be to have a metadata graph name per resource (as in TPF). I think the choice here is the right one, just making sure we're aware.
2. I'm not sure about the name; seems to me that all metadata is "generated" in some way. Maybe `ResponseMetadata`?

Reply:
I'm not sure exactly what you mean with "exposed". If you mean sending it as information somewhere in an HTTP response, the answer is no. Specifically:
 * Metadata always gets converted to headers, so there the graph gets lost in the process.
 * For containers the code that generates their representation has been updated to first force all metadata quads into the default graph before converting it to a data stream.

Never even thought about having a separate graph per resource. I don't think there would be any advantage to it since metadata is already separated per resource.

Reply:
All good.